python3/tests/test_raygunmsgs.py

```diff
@@ -268,7 +273,7 @@ def test_remove_larger_varaible(self):
         del globalReference
 
 
-class TestRaygunErrorMessage(unittest.TestCase):
+class TestRaygunErrorMessageChained(unittest.TestCase):
```

Comment:
This class name was duplicated, which caused that the above tests were ignored. This was detected thanks to the static analysis tool.